appeals/api/src/server/endpoints/appeals.d.ts

```diff
@@ -352,6 +352,7 @@ interface FolderInfo {
 
 interface LatestDocumentVersionInfo {
 	published: boolean | null | undefined;
+	draft: boolean;
```

Comment:
Out of curiosity, what is the new `draft` property for?

Reply:
Hey James, draft are the documents that don't have metadata yet... Settled for that to replace the current use of `published`.